content/departments/cloud/technical-docs/operations.md

```diff
@@ -204,14 +204,16 @@ Disk storage can be safely **increased** on managed instances at any time. Do no
 To increase the disk size:
 
 1. Set up your environment with `mi workon` as usual.
-1. Increase the value of `data_disk_size` in `terraform.tfvars` and run terraform apply
+1. Increase the value of `data_disk_size` in `terraform.tfvars
 1. Commit and push your changes:
 
    ```sh
    git add terraform.tfvars && git commit -m "$CUSTOMER: increase disk size"
    git push origin HEAD
    ```
 
+1. Once the PR is approved, merge the PR and then apply the changes on Terraform Cloud.
```

Comment:
add a notes saying this is only application to `rctest`, `sg` (that have TFC enabled)

Reply:
Good catch, I forgot this hadn't been rolled out to all instances.